src/framework/schema/custom-types.ts

```diff
@@ -0,0 +1,43 @@
+import * as NexusSchema from '@nexus/schema'
```

Comment:
This module is a bit of a hack, could benefit from a module level comment. In the future updated to link to the issue we'll create on schema component.